bears/general/InvalidLinkBear.py

```diff
@@ -44,7 +44,8 @@ def find_links_in_file(file, timeout, link_ignore_regex):
         link_ignore_regex = re.compile(link_ignore_regex)
         regex = re.compile(
             r'((ftp|http)s?://[^.:%\s_/?#[\]@\\]+\.(?:[^\s()%\'"`<>|\\]+|'
-            r'\([^\s()%\'"`<>|\\]*\))*)(?<!\.)(?<!,)')
+            r'\\]+|\([^\s()\'\"`<>|\\]*\))%?[0-9]*(?:[^\s()%\'\"`<>|\\]+|'
+            r'\([^\s()\'\"`<>|\\]*\))*)(?<!\.)(?<!,)')
```

Comment:
the thing is that if something just seems to work in the given testcases than it'll likely be buggy unless you understand fully *why* it works